frontend/.eslintrc.yml

```diff
@@ -0,0 +1,26 @@
+env:
+    browser: true
+    es2021: true
+extends:
+    - eslint:recommended
+    - plugin:@typescript-eslint/recommended
+overrides: []
+parser: '@typescript-eslint/parser'
+parserOptions:
+    ecmaVersion: latest
+    sourceType: module
+plugins:
+    - '@typescript-eslint'
+rules:
+    indent:
+        - error
+        - 2
```

Comment:
To be consistent with all our other projects this should be changed to 4

Reply:
@LindvedKrvang Rather we should have a talk about this. It is correct that we use 4 spaces for indentation in Sofie. Most of our other repos use 2 spaces or don't have it specified.

Reply:
Then we should align them to use 4 as well ;)